src/Bridge/Doctrine/Orm/Extension/FilterExtension.php

```diff
@@ -27,13 +27,15 @@
  */
 final class FilterExtension implements QueryCollectionExtensionInterface
 {
+    use FilterLocatorTrait;
+
     private $resourceMetadataFactory;
-    private $filters;
 
-    public function __construct(ResourceMetadataFactoryInterface $resourceMetadataFactory, FilterCollection $filters)
+    public function __construct(ResourceMetadataFactoryInterface $resourceMetadataFactory, $filterLocator)
```

Comment:
There is no such typing as `ContainerInterface | FilterCollection` in php right? Maybe we could document this new argument though.